Client/src/Validation/validation.js

```diff
@@ -128,10 +128,43 @@ const maintenanceWindowValidation = joi.object({
 	monitors: joi.array().min(1),
 });
 
+const advancedSettingsValidation = joi.object({
+	apiBaseUrl: joi.string().uri({ allowRelative: true }).trim().messages({
+		"string.empty": "API base url is required.",
+		"string.uri": "The URL you provided is not valid.",
+	}),
+	logLevel: joi.string().valid("debug", "none", "error", "warn").allow(""),
+	systemEmailHost: joi.string().allow(''),
+	systemEmailPort: joi.string().allow(''),
+	systemEmailAddress: joi
+		.string().allow(''),
+	systemEmailPassword: joi.string().allow(''),
+	jwtTTL: joi
+		.string()
+		.trim()
+		.messages({
+			"string.empty": "JWT TTL is required."			
+		}),
+	dbType: joi.string().trim().messages({
+		"string.empty": "DB type is required",
+	}),
+	redisHost: joi.string().trim().messages({
+		"string.empty": "Redis host is required",
+	}),
+	redisPort: joi.string().allow('').custom((value, helpers) => {
+		if(value && isNaN(parseInt(value))){
```

Comment:
Can we add custom messages to the other cases as well?

Reply:
jwt ttl custom validation is added :Resolved in : https://github.com/bluewave-labs/bluewave-uptime/commit/4966f3fd38e43960e2e6d3fbea78afb205c08883